examples/step-44/step-44.cc

```diff
@@ -2333,10 +2332,8 @@ namespace Step44
     // Next we assemble the Neumann contribution. We first check to see it the
     // cell face exists on a boundary on which a traction is applied and add
     // the contribution if this is the case.
-    for (unsigned int face = 0; face < GeometryInfo<dim>::faces_per_cell;
-         ++face)
-      if (cell->face(face)->at_boundary() == true &&
-          cell->face(face)->boundary_id() == 6)
+    for (const auto &face : cell->face_iterators())
+      if (face->at_boundary() == true && face->boundary_id() == 6)
```

Comment:
In almost all other places in this patch, we use the following instead:
```suggestion
      if (face->at_boundary() && face->boundary_id() == 6)
```
I don't care one way or the other, so let's just go with the common style.